gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -37,35 +64,31 @@ static const gchar *subplugin_prefixes[] = {
 };
 
 #define CONF_SOURCES (3)
+
 typedef struct
 {
-  gboolean loaded;            /**< TRUE if loaded at least once */
-  gboolean enable_envvar;     /**< TRUE to parse env variables */
-  gboolean enable_symlink;    /**< TRUE to allow symbolic link file */
-
-  gchar *conffile;            /**< Location of conf file. */
-
   /*************************************************
    * Cached Raw Values                             *
    * 0: ENVVAR. 1: CONFFILE, 2: Hardcoded          *
    *************************************************/
-  gchar *pathFILTERS[CONF_SOURCES];        /**< directory paths for FILTERS */
-  gchar *pathDECODERS[CONF_SOURCES];       /**< directory paths for DECODERS */
-  gchar *pathCUSTOM_FILTERS[CONF_SOURCES]; /**< directory paths for CUSTOM FILTERS */
-  gchar *pathCONVERTERS[CONF_SOURCES]; /**< directory paths for CUSTOM FILTERS */
+  gchar *path[CONF_SOURCES]; /**< directory paths */
```

Comment:
How about defining `enum CONF_SOURCES`? Then, line 350,369, and 370 can be revised to have straightforward meanings.

Reply:
Updated. Thanks!